lib/collibra/client/get_asset_responsibilities.ts

```diff
@@ -0,0 +1,18 @@
+import type { AxiosResponse } from "axios"
+import * as E from "fp-ts/Either"
+import * as TE from "fp-ts/TaskEither"
+import { pipe } from "fp-ts/function"
+
+type ResponsibilitiesResponse = AxiosResponse<Collibra.PagedResponsibilityResponse>
+const _getResponsibilities = (client: Net.Client, id: string) => () =>
+  client.get<ResponsibilitiesResponse>("/responsibilities", {
+    params: new URLSearchParams({ resourceIds: id }),
+  })
+
+export const getAssetResponsibilities =
+  (client: Net.Client) =>
+  (id: string): TE.TaskEither<Error, Collibra.Responsibility[]> =>
+    pipe(
+      TE.tryCatch(_getResponsibilities(client, id), E.toError),
```

Comment:
What's the underscore convention here, not used to these kind of conventions

Reply:
it was just because i was too lazy to find another name for the function 👀

Reply:
looks good....great to understand functional programming flow